ccmbioinfo/lfs
# CARDS-2244: Use TG-PMCC CARDIAC CLINICS as the clinic to import

To test:
- build with `mvn clean install -Pdocker`
- in `compose-cluster/mssql` run `python3 generate_test_proms_sql.py -n 50 proms_sample.sql`
- in `compose-cluster` run `python3 generate_compose_yaml.py --mongo_singular --dev_docker_image --cards_project cards4proms --composum --timezone America/Toronto --mssql --adminer`
- `docker-compose build && docker-compose up`
- at `http://localhost:1435/?mssql=mssql&username=sa&db=master&ns=path&import=` import the generated `proms_sample.sql` file
- open `http://localhost:8080/Subjects.importClarity`
- wait for about 2 minutes for the import to finish and all the Survey events to be generated
- check that there are about 45 patients imported
- check that the patients that had `TG-PMCC CARDIAC CLINICS DIAGNOSTIC` as the clinic in the sample sql file were imported with the `TG-PMCC CARDIAC CLINICS` clinic and have the 5 surveys generated
